components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/java/org/wso2/micro/integrator/management/apis/LoggingResource.java

```diff
@@ -114,24 +148,41 @@ public boolean invoke(MessageContext messageContext) {
         return true;
     }
 
+    private JSONObject updateLoggerData(org.apache.axis2.context.MessageContext axis2MessageContext, String loggerName,
+                                        String loggerClass, String logLevel) {
+
+        try {
+            updateConfigs();
+            String modifiedLogger = getLoggers().concat(", ").concat(loggerName);
+            updateLog4j2Property(LOGGERS_PROPERTY, modifiedLogger);
+            updateLog4j2Property(LOGGER_PREFIX + loggerName + LOGGER_NAME_SUFFIX, loggerClass);
+            updateLog4j2Property(LOGGER_PREFIX + loggerName + LOGGER_LEVEL_SUFFIX, logLevel);
+            applyConfigs();
+            jsonBody.put(Constants.MESSAGE,
+                         "Successfully added logger for ('" + loggerClass + "') with level " + logLevel);
```

Comment:
Shall we add the logger name to as well?

Reply:
I intentionally left it since it is too much of information for the api call back and it just a variable user defined and has no real impact as in the case of class. wdyt

Reply:
Seems okay, but when we update the log-level of a logger we send both the logger name and log level in the response message. It would be better if we have consistency among these messages. WDYT?

Reply:
added in https://github.com/wso2/micro-integrator/pull/1723/commits/ef4dae2149efe634106dabcd05f5ef1cbf4c624e